src/de/emodelle.html

```diff
@@ -1,5 +1,15 @@
-
-      <%= require('html-loader!./components/02_footer_section/index.html').default %>
-
-      <br>
-      THIS VARIABLE IS FROM EMODELLE:   <%= htmlWebpackPlugin.options.newVariable %>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+      <head>
+            <!-- relative path to SCSS source file -->
+            <link href="./components/02_footer_section/style.scss" rel="stylesheet" />
```

Comment:
you can define source style file directly in the HTML template using the `link` tag and the path to source SCSS file